contents/testing/writing-testable-code.md

```diff
@@ -0,0 +1,110 @@
+# Writing Testable Code
+
+Below is a collection of 10 rules that you can follow to write testable code.
```

Comment:
Thanks for the detailed review! I'll reformat the 10 rules under four categories (i.e the four reasons I used in lightningtalks)